src/main/java/com/rcv/FileUtils.java

```diff
@@ -45,6 +46,8 @@ static void createOutputDirectory(String dir) throws UnableToCreateDirectoryExce
       // dirFile is the File object for dir
       File dirFile = new File(dir);
       if (!dirFile.exists() && !dirFile.mkdirs()) {
+        Logger.log(Level.SEVERE, "Failed to create output directory: %s", dir);
+        Logger.log(Level.SEVERE, "Check the directory name and permissions.");
```

Comment:
Combine with line breaks here and any other cases where we have multiple lines spitting out like this.

Reply:
I actually like this style better.  It's more readable.  Is there some advantage to doing this?

Reply:
Yeah, sorry comments got messed up in GH; this was regarding certain log parsers that count number of errors. Looks like it made sense to you below so you fixed, thanks.